tests/php80-compatibility.patch

```diff
@@ -226,3 +336,16 @@ index 6033eca..5e26ed8 100644
 -    private $x = 1;
 +    private int|string|null $x = 1;
  }
+diff --git a/tests/input/TralingCommaOnFunctions.php b/tests/input/TralingCommaOnFunctions.php
+index ed304c9..e383ee2 100644
+--- a/tests/input/TralingCommaOnFunctions.php
++++ b/tests/input/TralingCommaOnFunctions.php
+@@ -6,7 +6,7 @@ namespace Doctrine;
+ 
+ class TralingCommaOnFunctions
+ {
+-    public function a(int $arg): void
++    public function a(int $arg,): void
```

Comment:
Hmm, what's going on here?

Reply:
I had to patch test inputs for php 8.

Reply:
Makes sense: I'm always a proponent for dropping older PHP versions (this year, PHP 7 dies anyway), but I understand the lack of energy to fight uphill against a different audience :+1: